vm/systemSmartContracts/eei.go

```diff
@@ -417,16 +508,16 @@ func (host *vmContext) ExecuteOnDestContext(destination []byte, sender []byte, v
 		return nil, err
 	}
 
-	err = host.Transfer(callInput.RecipientAddr, callInput.CallerAddr, callInput.CallValue, nil, 0)
+	err = host.transferBeforeInternalExec(callInput, sender)
```

Comment:
is this still backward compatible? On the original code the sender was `callInput.CallerAddr`.

Reply:
callInput.CallerAddr == sender